rhizome.js

```diff
@@ -67,6 +67,10 @@ var zoom =
     g.attr("transform", "translate(" + d3.event.transform.x + "," +
           d3.event.transform.y + ")scale(" + d3.event.transform.k + ")");
   });
+
+var link_colour_default = '#999';
```

Comment:
these should probably be `linkColourDefault` instead of `link_colour_default` but it's so hideous...

Reply:
JavaScript.